create/how_to/aws.md

```diff
@@ -120,7 +121,7 @@ Configuring your MeiliSearch instance in a production environment is not just st
 
 If you want to use your own domain name (or sub-domain), add an `A record` in your domain name provider account. Otherwise, **you can skip this step**.
 
-![Domain to  MeiliSearch](/aws/11.domain.png)
+![my-aws-instance domain](/aws/11.domain.png)
```

Comment:
Agreed. Idk why I used "fragment". Not sure what would be better, I don’t like part. Any suggestions are welcome

Reply:
🤔  fragment was my (not so great) suggestion. 

What I meant was that `my-aws-instance domain` was fairly opaque as a description to an image. Like, this as an alt text seems very difficult to understand because it says what is in one box of the page, but gives no explanation on what it is or where it goes.

You could potentially make do without saying a fragment and just focus on what this interface does/is showing. Something like "An interface for editing DNS records. The A record is set to my-aws-instance."

Reply:
🤦🏻‍♀️ My bad. I like "An interface for editing DNS records. The A record is set to my-aws-instance."